### ISSUE TITLE
[BUG] Cannot find name 'ADMONITION_TYPES'

### ISSUE THREAD

**Issue by joelBlixify** [mdx-editor/editor#524]
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] I have read the documentation and cannot find an answer.

**Describe the bug**
Unable to compile and deploy due to this bug, where ADMONITION_TYPES is not declared, understandably due to it being excluded from this release, however, the error prevents deployment due to the git workflow failing.

**Reproduction**
Quite simply, in NextJS, using TypeScript and Yarn, running yarn format will prompt the error, and also git workflows will fail.

**Expected behavior**
Remove usage of the undeclared ADMONITION_TYPES variable, or declare the variable to remove the error.

**Screenshots**
![image](https://github.com/mdx-editor/editor/assets/122084887/43841a7e-fc96-4f7d-a804-53821435cb41)

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS, Windows] - Windows can act funny sometimes.
 - Browser [e.g. Chrome, Safari] - Chrome.
 - IDE - VSCode
 - VCS - GitHub

**Additional context**
Hope there is an easy fix for this so that you can deploy a new version as effortlessly as possible, or any suggestion if there is a workaround to this is very welcome as well.

**Comment by joelBlixify** [comment]
If anyone else is facing a similar issue to mine, you may try this as a solution:
![image](https://github.com/mdx-editor/editor/assets/122084887/2345a2c8-3637-4b64-98b0-a2ef351a4dd0)

Importing the library this way bypasses the type checking

**Comment by petyosi** [comment]
My guess is that you have `skipLibCheck` set to false?

**Comment by joelBlixify** [comment]
@petyosi Correct, unfortunately we are required to set it to false to ensure that there are no errors coming from the various other libraries that we are using.

**Comment by petyosi** [comment]
I'll look into fixing this, it should be fairly straight forward.

**Comment by eduardoliron** [comment]
Facing the same issue.

Great repo, by the way. Thanks for your effort!
